### DISCUSSION TITLE
Enable GitHub sponsorship in Ukraine

### DISCUSSION THREAD

**Discussion by qwerty541** [community/community#67578] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Recently one person expressed interest in sponsoring my open source contributions and i found out that unfortunately this feature is not available in my country. Ukraine is going through difficult times right now, most citizens have problems with finances and work, and I think that enabling this feature would be an excellent measure of support. I'm a bit surprised that no one made such request before or may be i'm blind.

**Comment by GhostOf0days** [comment]
You could use alternative platforms like Buy Me a Coffee or Patreon in the meantime.

**Reply by qwerty541** [reply]
Thanks for the tip. Before i created this discussion i thought that this might be the only solution for now. I created this discussion since i assume that GitHub choose next countries to enable sponsorship feature depending on demand. It is an opportunity for people to show their interest. If everyone remains silent on this, then this function will be enabled in my country as the last thing.

**Reply by dukhevych** [reply]
> You could use alternative platforms like Buy Me a Coffee or Patreon in the meantime.

Patreon allows only subscriptions. Buymeacoffee shut down operations in Ukraine. Ko-fi also doesn't work for Ukraine.
There's literally no one-time-donations service available for Ukraine currently (apart from local ones focused on Ukrainian market).

So GitHub sponsorship is really needed here.

**Comment by dabakovich** [comment]
I'm also interested in some approach to sponsoring. Unfortunately, Buymeacoffee no longer supports withdrawals into Ukraine. GitHub Sponsors appeared to me as a perfect native solution, but it doesn't support Ukraine as well.

Furthermore, guys @qwerty541, @dukhevych, what alternative did you found for sponsoring?

**Comment by punyflash** [comment]
There are no ways to get sponsored in Ukraine right now except direct payments. Would be nice to have at least some option

**Comment by globart** [comment]
@dabakovich @punyflash @dukhevych @qwerty541 you could try [Base by Mono](https://base.monobank.ua/)

**Reply by punyflash** [reply]
You can't create base without having telegram/youtube/instagram/twitch/tiktok channel and meant to collect dotations on streams. There is no relations to github

**Reply by globart** [reply]
In this case, perhaps [Donatello](https://donatello.to/) will be suitable in the meantime
